### DISCUSSION TITLE
Migrate all things to another PC

### DISCUSSION THREAD

**Discussion by Ibuup** [stashapp/stash#5098] [Q&A] [answered]
Hi, thank you for development of this nice tool.
I have a question about migrating all files of stash database to my another pc.

I think the app file (.exe) can be just installed on the target pc from this repo.

But how can I migrate my personal DBs?
(Generated files, manually set thumbnails of scenes/tags, app settings, tracked activities, etc)

Thank you.

**Comment by HereIam80** [comment, marked as answer]
These are the steps you need to take:

- Backup your existing database.
- Move/copy your blobs and generated folders to the new destination if necessary.
- Install stash on the new machine (docker or otherwise).
- Point your blobs and generated paths for the correct folders.
- Point your library to the folders with your content like you had before.
- Once Stash is set up (and still empty), close stash/the docker container.
- Also close your Stash of your previous installation and copy over your stash-go.sqlite file from your old install to your new installation, replacing the existing (empty) database.
- Start stash and see if everything is there.
- If the paths are different now than they were before, run a scan task.

The scan will update any paths from the previous installation (like when you are changing from a windows to a linux/docker installation)

**Reply by handwork3** [reply]
this worked perfectly for migration from win 10 to macOS 15.5.  Thanks!!!!
